CommunityFridgeMapApi/tests/unit/test_tag.py

```diff
@@ -0,0 +1,57 @@
+from multiprocessing import connection
+
+from dependencies.python.db import layer_test
+from dependencies.python.db import get_ddb_connection
+import unittest
+from dependencies.python.db import Tag
+
+def test_layer_test():
```

Comment:
don't have to test this, it's already tested in the `test_fridge.py` file

I should probably not have the test there to be honest! I'll move it to a different test file at some point :)